app/javascript/components/Payment/PaymentTypeSelection.js

```diff
@@ -12,8 +12,25 @@ export class PaymentTypeSelection extends PureComponent {
     return true;
   }
 
+  // this is done since PAYPAL doesnt support ARS currency as of now
+  showPaymentType(type, currency) {
```

Comment:
What about expressing this with a conditional operator?  `return currency === 'ARS' && type === 'paypal' ? false : true;`